completions/_kbsecret

```diff
@@ -20,12 +20,11 @@ case "$state" in
       'help:print command-line help'
       'list:list records'
       'login:access login records'
-      'new-session:create a new session'
       'new:create a new record'
       'pass:retrieve password from a login record'
       'raw-edit:edit the JSON of a record'
-      'rm-session:desconfigure and delete a session'
       'rm:remove a record'
+      'session:Manage sessions'
```

Comment:
Lowercase "Manage", just for consistency.

Reply:
Done